planet/src/main/webapp/js/itinerary.js

```diff
@@ -7,6 +7,20 @@ function closeForm() {
     document.getElementById('add-event').style.display = 'none';
 }
 
+function handleHotelChange() {
+    if (typeof(Storage) !== "undefined") {
+        sessionStorage.setItem('hotel', document.getElementById('hotel-address').value);
+    } else {
+        alert ('Please update your browser'); 
+    }
+}
+
+function renderHotel() {
```

Comment:
nit: as we discussed previously, I think we should rename all the "hotel" variables to a starting point (the user might not start at a hotel) ex: startLocation

Reply:
Yes! I will fix that.